File: pgs_완주하지_못한_선수.py
def solution(participant, completion):
    participant.sort()
    completion.sort()
    answer = list()
    for i in range(len(participant)):
        if i == len(participant)-1:
            answer.append(participant[i])
        elif participant[i] != completion[i]:
            answer.append(participant[i])
            break
    
    
        
    
    return answer[0]

File: JS_Algo/pgs_완주하지_못한_선수.js
function solution(participant, completion) {
  participant = participant.sort();
  completion = completion.sort();

  for (let i = 0; i < completion.length; i++) {
    if (participant[i] !== completion[i]) {
      return participant[i];
    }
  }
  return participant[participant.length - 1];
}
// p : 12345
// c : 1234
// 참가자 배열, 완주자 배열 정렬 =>
//
